cmd/cluster/clusterCreate.go

```diff
@@ -111,28 +111,28 @@ func NewCmdClusterCreate() *cobra.Command {
 	/*********
 	 * Flags *
 	 *********/
-	cmd.Flags().StringP("api-port", "a", "random", "Specify the Kubernetes API server port exposed on the LoadBalancer (Format: `--api-port [HOST:]HOSTPORT`)\n - Example: `k3d create -m 3 -a 0.0.0.0:6550`")
-	cmd.Flags().IntP("masters", "m", 1, "Specify how many masters you want to create")
-	cmd.Flags().IntP("workers", "w", 0, "Specify how many workers you want to create")
+	cmd.Flags().String("api-port", "random", "Specify the Kubernetes API server port exposed on the LoadBalancer (Format: `--api-port [HOST:]HOSTPORT`)\n - Example: `k3d create -m 3 -a 0.0.0.0:6550`")
```

Comment:
@iwilltry42 isn't the example `-a` now wrong?

Reply:
from the RC docs:
> Important: Due to this renaming, the short-hand flag k3d cluster create -a now stands for --agents and not anymore for --api-port (which does not have a short-hand flag anymore)

Reply:
Yep, you're right, didn't catch this, thank you 👍

Reply:
Fixed now :+1: